### DISCUSSION TITLE
Why is Total Commander "stuck" at version 10.0 on chocolatey?

### DISCUSSION THREAD

**Discussion by gobalius** [chocolatey-community/chocolatey-packages#2077] [General]
Total Commander package is "stuck" at version 10.0.0.20210903
I can see a discussion about the installation path and there was a bug, which got fixed, as far as I understand it. The package is automatically maintained (AU), the files in the repo get updated (they show 10.52 as the latest version), however, when doing choco upgrade totalcommander it says I already have the latest 10.0.0.20210903

**Comment by RedBaron2** [comment]
@gobalius 
It would appear that something needs to be done to fix this being approved by the automated system for the chocolatey community feed.
Maybe @gep13, @AdmiringWorm, or @pauby might be able to find out why this package fails to be approved for the community feed.

**Reply by pauby** [reply]
@RedBaron2 @gobalius The package failed validation as can be seen from the [(latest) failed version](https://community.chocolatey.org/packages/TotalCommander/10.52).

**Reply by gobalius** [reply]
The validation failing confuses me... it says binary files are included in the package, but I don't see any in the package folder in the repo...

**Reply by pauby** [reply]
@gobalius Can you point to where it says that so I can look?

**Reply by gobalius** [reply]
@pauby on the link you provided, it says, quote:

> This package was rejected on 30 Nov 2022. The reviewer chocolatey-ops has listed the following reason(s):
>     Binary files (.exe, .msi, .zip) have been included. The reviewer will ensure the maintainers have distribution rights. More...

**Reply by gobalius** [reply]
Oh... it was an unfortunate scroll... the first line I copied and the second one are not supposed to be together... I got this totally wrong. I still don't exactly know what is wrong, but it's not what it seemed to me before.

**Comment by pauby** [comment]
It is failing Package Validator for this reason:

![image](https://user-images.githubusercontent.com/12760779/208263446-bf314dcf-9bca-465e-af2d-dbdb694369a1.png)

A quick check looks like it is likely [this](https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/totalcommander/tools/chocolateyInstall.ps1#L8) and maybe also [this](https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/totalcommander/tools/chocolateyInstall.ps1#LL9-L9C31).

**Reply by TheCakeIsNaOH** [reply]
The reason behind the usage of `$env:chocolateyInstallArguments` is due to this bug: https://github.com/chocolatey/choco/issues/2345
